src/cclib/parser/data.py

```diff
@@ -70,6 +70,7 @@ class ccData(object):
         scftargets -- targets for convergence of the SCF (array[2])
         scfvalues -- current values for convergence of the SCF (list of arrays[2])
         temperature -- temperature used for Thermochemistry (float, kelvin)
+        time -- time in molecular dynamic trajectories (array[1], fs)

```

Comment:
I'm OK with adding this attribute, but perhaps change description to "time in molecular dynamics and other trajectories" since we could use this for any other models of dynamics.